lib/std/Target/aarch64.zig

```diff
@@ -2045,7 +2045,7 @@ pub const cpu = struct {
             .sme2,
             .sme_f64f64,
             .sme_i16i64,
-            .v9_2a,
+            .v8_7a,
```

Comment:
Yes, this is intentional: https://github.com/alexrp/llvm-project/commit/eba1ef5a1b7a84ed1954797dcd6d6f073b1f1a56